### ISSUE TITLE
Get Static VS Analysis Working in OneBranch

### ISSUE THREAD

**Issue by nibanks** [microsoft/undocked#6]
The props file currently disables static VS analysis when running in the official build. We need to work with the necessary team(s) to get this fixed.

**Comment by mtfriesen** [comment]
@nibanks is this still a problem? I believe XDP has been building with static analysis.

**Comment by nibanks** [comment]
We only set `<RunCodeAnalysis>true</RunCodeAnalysis>` when not in OneBranch.

**Comment by mtfriesen** [comment]
> We only set `<RunCodeAnalysis>true</RunCodeAnalysis>` when not in OneBranch.

Yep, I'm asking if we still need to make that conditional. In XDP we had been setting RunCodeAnalysis unconditionally, and not hitting issues in OneBranch.

**Comment by nibanks** [comment]
I'd love to remove this then. Let's try it.
